R/random-forest-development.R

```diff
@@ -461,6 +486,35 @@ RandomForestDevelopment <- R6Class("RandomForestDevelopment",
     getCutOffs = function() {
       warning("`getCutOffs` is deprecated. Please use `generateAUC` instead. See 
               ?generateAUC", call. = FALSE)
+    },
+    
+    getVariableImportanceList = function(numTopVariables = NULL) {
+      top <- min(numTopVariables, nrow(private$variableImportanceList))
+      return(private$variableImportanceList[1:top, ])
+    },
+    
+    plotVariableImportance = function(numTopVariables = NULL) {
```

Comment:
I found an issue with the y-axis text if the feature names are long. See below:
![image](https://user-images.githubusercontent.com/15145347/29690824-c7541586-88e5-11e7-88f4-f5c017cfa404.png)